lib/elixir/lib/enum.ex

```diff
@@ -2046,6 +2066,24 @@ defmodule Enum do
   end
 
   @doc """
+  Sorts the collection, removing all duplicated items.
```

Comment:
See comment above, but also run doesn't work here IMO, since run implies consecutive duplicates, which is what usort does differently from dedup.